### ISSUE TITLE
No description detected?

### ISSUE THREAD

**Issue by fosskers** [ultralisp/ultralisp#219]
Hi there, thanks again for the recent updates. 

I recently added a new package from Source Hut, but noticed that its description doesn't seem to appear on [its Ultralisp page](https://ultralisp.org/projects/fosskers/fn-macro).

Note that I have a proper description both in my `.asd` and in my `defpackage`, like:
```lisp
(defsystem "fn-macro"
  :version "1.0.0"
  :author "Colin Woodbury"
  :mailto "colin@fosskers.ca"
  :license "GPL-3.0-only"
  :depends-on ()
  :components ((:module "src"
                :components
                ((:file "fn-macro"))))
  :description "A convenient lambda shorthand."
  :in-order-to ((test-op (test-op "fn-macro/tests"))))
```
and
```lisp
(defpackage fn-macro
  (:use :cl)
  (:export #:fn)
  (:documentation "A convenient lambda shorthand."))
```
Idea any of what is happening here? Is it possible that the description is coming from the original _Github_ description field and not the system definition? 

Thanks!

**Comment by svetlyak40wt** [comment]
You are correct. For github projects description is fetched via API whereas for Git projects I can't do that.

Loading asd systems only to load description seems to complex for me right now, because Ultralisp loads system code in separate docker containers running "check" tasks. To retrieve description I need either to modify these checkers or to add another kind of tasks.

That is why as a first more I want to make any project's description editable. This way you'll be able to enter any description you want.

Probably I'll be able to add automatic description fetch from asd files later, when will work on this issue https://github.com/ultralisp/ultralisp-client/issues/1 about additional systems information such as licenses, readme, etc.

**Comment by fosskers** [comment]
> That is why as a first more I want to make any project's description editable.

That would do it! But no rush :pray: Thanks again.
